storage-logzio/src/test/java/zipkin2/storage/logzio/LogzioSpanStoreTest.java

```diff
@@ -1,49 +1,38 @@
 package zipkin2.storage.logzio;
 
+import okhttp3.Headers;
 import okhttp3.mockwebserver.MockResponse;
 import okhttp3.mockwebserver.MockWebServer;
-import okhttp3.mockwebserver.RecordedRequest;
-import org.junit.AfterClass;
-import org.junit.Assert;
-import org.junit.BeforeClass;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import zipkin2.Span;
 import zipkin2.storage.logzio.client.HttpCall;
 import zipkin2.storage.logzio.client.SearchCallFactory;
 import zipkin2.storage.logzio.client.SearchRequest;
 
-import java.io.IOException;
 import java.util.List;
 
+import static org.assertj.core.api.Assertions.assertThat;
 
 public class LogzioSpanStoreTest {
-    private static MockWebServer mockWebServer = new MockWebServer();
-
-    private static final Logger logger = LoggerFactory.getLogger(LogzioSpanStoreTest.class);
+    @Rule public MockWebServer mockWebServer = new MockWebServer();
 
     private static String apiToken = "not-a-real-api-token";
     private static LogzioStorageParams params = new LogzioStorageParams();
     private static LogzioStorage storage;
     private static LogzioSpanStore spanStore;
 
-    @BeforeClass
-    public static void setup() {
-        logger.info(LogzioStorage.ZIPKIN_LOGZIO_STORAGE_MSG + "Setting up test environment");
```

Comment:
by default, the junit rule already logs status messages. there's no need to do that again here, knowing this.